### DISCUSSION TITLE
Certain jobs have been pending for execution for over 24 hours.

### DISCUSSION THREAD

**Discussion by viceice** [goharbor/harbor#19905] [Q&A] [answered]
I'm seeing the message `Certain jobs have been pending for execution for over 24 hours. Please check the job service dashboard.
For more details, please refer to the Wiki.` on my limited account.
When i login with an admin account i don't see that message and also no pending jobs.

How can i find the pending jobs?

**Comment by wy65701436** [comment]
what version of harbor are you using? If it is higher than v2.7, you can view the pending jobs in the jobservice dashboard. cc @stonezdj

**Reply by viceice** [reply]
I'm using `v2.10.0-6abb4ea` and there is no pending job when logging in as administrator. the normal user get's the message but going to the dashboard isn't allowed.

**Reply by MinerYang** [reply]
cc @stonezdj 
Will transfer as an issue

**Reply by FelixRodriguezJara** [reply]
I am having the exact same issue. Is there any update on it?

**Comment by ivoistaken** [comment]
We are experiencing the same with harbor 2.11.0

The issue happens only for non-admins, it might be connected with the following:
GET /api/v2.0/jobservice/queues
200 + json for admins
403 for non-admins - may be this triggers the yellow warning?

**Comment by blankdots** [comment]
seems there is a bug but it auto-close https://github.com/goharbor/harbor/issues/19923 ... maybe someone can re-open it

**Comment by luidoc** [comment]
Could someone please reopen issue #19923 or look for a solution to the problem in this issue?

**Comment by viceice** [comment, marked as answer]
- #19923
